newIDE/app/src/UI/Theme/ShadesofPurpleTheme/EventsSheet.css

```diff
@@ -0,0 +1,218 @@
+/**
+ * This is the styling applied to an EventsSheet.
+ * This also overwrite the default react-sortable-tree css.
+ */
+
+ .gd-events-sheet-Shades-of-Purple-theme.events-tree {
+  /* Use the fonts provided by the operating system(s) as possible. */
+  /* If you update this font list, be sure to do it in all the other places using fonts in the codebase. */
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto,
+    Oxygen-Sans, Ubuntu, 'Helvetica Neue', Helvetica, sans-serif,
+    'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol' !important;
+  font-size: 14px !important;
+  background-color: #2D2B55;
+}
+
+/**
+ * Ensure scroll is only on Y direction, preventing the display
+ * of horizontal scrollbar.
+ */
+.gd-events-sheet-Shades-of-Purple-theme .rst__virtualScrollOverride {
+  overflow-x: hidden !important;
+}
+
+/**
+ * Draggable handle on the left of an event
+ */
+.gd-events-sheet-Shades-of-Purple-theme .rst__moveHandle {
+  background: #4D21FC !important;
+}
+
+.gd-events-sheet-Shades-of-Purple-theme .rst__moveHandle:hover {
+  background-color: #FAD000 !important;
+}
+
+/**
+ * Scaffolding lines on the left
+ */
+.gd-events-sheet-Shades-of-Purple-theme .rst__lineHalfHorizontalRight::before,
+.gd-events-sheet-Shades-of-Purple-theme .rst__lineFullVertical::after,
+.gd-events-sheet-Shades-of-Purple-theme .rst__lineHalfVerticalTop::after,
+.gd-events-sheet-Shades-of-Purple-theme .rst__lineHalfVerticalBottom::after {
+  background-color: #9999E9;
```

Comment:
I don't see this color in the theme.
https://github.com/ahmadawais/shades-of-purple-vscode#sops-syntax-colors

Where is it used in GDevelop?